src/server/hotkey_collector.cpp

```diff
@@ -21,16 +21,24 @@
 #include <dsn/utility/smart_pointers.h>
 #include "base/pegasus_key_schema.h"
 #include <dsn/dist/fmt_logging.h>
+#include <dsn/utility/flags.h>
 
 namespace pegasus {
 namespace server {
 
+DSN_DEFINE_int32(
+    "pegasus.server",
+    hotkey_collector_max_work_time,
```

Comment:
How about `max_seconds_to_detect_hotkey` or something like that, unit should be present for admins.